addons/main/functions/fnc_drawDownedUnitIndicator.sqf

```diff
@@ -1,17 +1,24 @@
 #include "script_component.hpp"
 
-private _size = GVAR(showDownedUnitIndicatorSize) * ((call CBA_fnc_getFov) select 1);
+private _zoom = (call CBA_fnc_getFov) select 1;
+private _size = GVAR(showDownedUnitIndicatorSize) * _zoom;
```

Comment:
Why split zoom and user input for size? Imo that should not be.

Reply:
Because zoom is used 3 times in the function, so rather than call the CBA function 3 times I just put it in a variable.

Reply:
?
It was more meant why have the zoom variable anyways. The user can change the size of the icon on the fly, imo the timer size should corelate to that as well.